2024-08-24 17:39:41,787 [INFO]  starting up 24-08-24-1739_setting0 at 12.682706432999112
2024-08-24 17:39:41,840 [INFO]  git SHA: a2a5c43fb29217d9d662423e1a729b55dd49a308
2024-08-24 17:39:41,841 [INFO]  setting file name: s0
2024-08-24 17:39:41,842 [INFO]  parameters {'points_per_cent': 2, 'time_limit': 5, 'correct_responses': ['f', 'j'], 'combination_num': 8}
2024-08-24 17:39:43,756 [INFO]  Created window with size [1300 1000]
2024-08-24 17:39:43,953 [INFO]  Using dummy triggers
2024-08-24 17:39:43,953 [INFO]  begin main
2024-08-24 17:39:43,954 [DEBUG]  message: Ready? Let go! (None)
2024-08-24 17:49:48,712 [ERROR]  Caught exception in stage main
Traceback (most recent call last):
  File "/Users/wangying/Desktop/PupilDilationPsychopy/PupilDilationExp/experiment.py", line 40, in wrapper
    f(self, *args, **kwargs)
  File "/Users/wangying/Desktop/PupilDilationPsychopy/PupilDilationExp/experiment.py", line 535, in main
    self.message(msg= "Ready? Let go!", space= True)
  File "/Users/wangying/Desktop/PupilDilationPsychopy/PupilDilationExp/experiment.py", line 267, in message
    event.waitKeys(keyList=['space', KEY_CONTINUE])
  File "/Users/wangying/Desktop/PupilDilationPsychopy/myenv/lib/python3.9/site-packages/psychopy/event.py", line 529, in waitKeys
    keys = getKeys(keyList=keyList, modifiers=modifiers,
  File "/Users/wangying/Desktop/PupilDilationPsychopy/myenv/lib/python3.9/site-packages/psychopy/event.py", line 438, in getKeys
    if key[0] in keyList:
KeyboardInterrupt
